livedata/build.gradle

```diff
@@ -23,17 +23,15 @@ jacocoAndroidUnitTestReport {
     csv.enabled false
     html.enabled true
     xml.enabled true
-    excludes += ['**/*_Impl.java',
-                 '**/*JavascriptBridge.class']
+    excludes += ['**/*_Impl.java', '**/*JavascriptBridge.class']
 }
 
-
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
     buildToolsVersion "29.0.3"
     defaultConfig {
         minSdkVersion 21
-        targetSdkVersion 28
```

Comment:
The SDK is not ready to be targeted to Android SDK Version 29 yet, we need to do some changes previously

Reply:
@JcMinarro, it is now addressed by putting the version 28 again.
Thank you for reviewing.

30c5fa4